content/en/real_user_monitoring/mobile_and_tv_monitoring/flutter/setup.md

```diff
@@ -279,11 +281,12 @@ MaterialApp.router(
 )
 ```
 
-For examples that use routers other than `go_router`, see [Advanced Configuration - Automatic View Tracking][7].
+For examples that use routers other than `go_router`, see [Automatically track views][11].
+
 
 ### Renaming Views
 
-For all setups, you can rename views or supply custom paths by providing a [`viewInfoExtractor`][8] callback. This function can fall back to the default behavior of the observer by calling `defaultViewInfoExtractor`. For example:
+For all setups, you can rename views or supply custom paths by providing a [`viewInfoExtractor`][12] callback. This function can fall back to the default behavior of the observer by calling `defaultViewInfoExtractor`. For example:
```

Comment:
```suggestion
For all setups, you can rename views or supply custom paths by providing a [`viewInfoExtractor`][5] callback. This function can fall back to the default behavior of the observer by calling `defaultViewInfoExtractor`. For example:
```

Reply:
same with this one, according to your links below, 5 links to the info extractor